pkg/linguist/dev.go

```diff
@@ -66,13 +66,15 @@ func GetDevConfig(language string) *model.Dev {
 	return dev
 }
 
-func normalizeLanguage(language string) string {
+func getPlatform(language string) string {
```

Comment:
None of these but node is a `platform`. I think this is more confusing that the older naming scheme.